sirepo/sim_data/opal.py

```diff
@@ -34,6 +34,10 @@ def fixup_old_data(cls, data):
             if cmd._type == "filter":
                 cmd.type = cmd.type.upper()
             elif cmd._type == "particlematterinteraction":
+                if cmd.type not in [
```

Comment:
What do you think of this fixup @moellep ?

Reply:
Yes, that looks good. It doesn't look like particlematterinteraction is a very commonly used feature.